schemas/recipes_schema.json

```diff
@@ -2,11 +2,10 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "recipes",
   "description":"A dictionary of quoted numerical item IDs. Each ID maps to a list of recipes. There may be multiple different recipes per item (same ID and metadata). The recipes may not be sorted.",
-
   "definitions": {
     "id": {
-      "type":["integer","null"],
-      "description":"A single numerical ID or null."
+      "type":["string", "integer","null"],
```

Comment:
why not just continue using integer ids ?

Reply:
The game doesn't anymore, why should we?

Reply:
yes it does ? the protocol still has integer ids everywhere.

Reply:
Okay, I get it. But for recipes, does it really matter?

Reply:
Not sure what you mean. No it doesn't matter to use one or the other.
However it matters to have consistent files between versions or else it means having separate implementations for each version which is not great.
That doesn't seem too hard to convert recipes to integer ids ?

Reply:
The game is now using text ids, and so was in 1.12, using text ids makes the files way more readable. Also the recipes don't use numeric ids anywhere in the vanilla game

Reply:
Good point but really that file is not highly readable anyway so I'm not sure if we should change this...

Reply:
Fixed